internal/place/client.go

```diff
@@ -141,6 +142,10 @@ func (a Address) String() string {
 	return strings.Join(a.Lines(), ", ")
 }
 
+func (a Address) HTML() template.HTML {
```

Comment:
with the separate template, could this be removed and the existing `address-lines` one be used?